pkg/k8shandler/secret_test.go

```diff
@@ -0,0 +1,67 @@
+package k8shandler
+
+import (
+	"reflect"
+	"testing"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+    rbac "k8s.io/api/rbac/v1"
+
+    "sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestClusterLoggingRequest_CreateOrUpdateSecret(t *testing.T) {
+
+    clusterRoleBinding := &rbac.ClusterRoleBinding{}
+
+    clusterLoggingRequest := &ClusterLoggingRequest{
+        client:  fake.NewFakeClient(clusterRoleBinding),
+        cluster: &logging.ClusterLogging{},
+    }
+
+	// mimic ca.crt
+	caCrtStr := `-----BEGIN CERTIFICATE-----
```

Comment:
I don't think you need a "realistic" cert/key - something shorter like `this is a cert` or `this is a key` instead of the long base64 encoded value - I think having a multiline string is good